web/wp-content/plugins/tmbi-nativo/class-tmbi-nativo.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * TMBI Nativo
+ *
+ * @package TMBI Nativo
+Plugin Name: TMBI Nativo
+Version: 1.0
+Description: Adds the nativo script to the header for TOH,FHM <a href='https://readersdigest.atlassian.net/browse/WPDT-4699' target='_blank'>Read more at WPDT-4699 ...</a>
+Author: DRDT Team
+Text Domain: tmbi-nativo
+License: BSD(3 Clause)
+License URI: http://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Including file for the settings.
+ *
+ * @file
+ */
+require 'inc/class-tmbi-nativo-in-articles.php';
+
+/**
+ *  Class TMBI_Nativo.
+ */
+class TMBI_Nativo {
+	const VERSION            = '1.0';
+	const NATIVO_CUSTOM_TYPE = 'nativo_custom_post';
+
+	/**
+	 * Add hooks and filters for this module.
+	 *
+	 * @action init
+	 */
+	public function __construct() {
```

Comment:
I think there's no need to instantiate the class if all the action happens here in the constructor. I've seen how instantiating the class results in extra opcode, compared to just adding the filters outside the class (and referencing the functions like `array( 'TMBI_Nativo', 'register_nativo_post_type' )`) but I'm not sure if it has a performance impact. I'll create a ticket to review and confirm!